client/src/explorer_common/resource_explorer_common.js

```diff
@@ -115,6 +114,16 @@ export const get_col_defs = ({ year }) => [
     get_val: (node) => _.get(node, "data.resources.spending"),
     val_display: (val) =>
       _.isNumber(val) ? <Format type="compact1" content={val} /> : null,
+    first_node_m2m_display: (val) =>
+      _.isNumber(val) ? (
+        <Format
+          type="compact1"
+          content={Math.ceil(val / 10000000) * 10000000}
```

Comment:
The `compact1` formatter applies rounding, why the extra rounding here?

Reply:
I think we wanted to have extra rounding to reduce the perceived precision of the estimation